commercetools-models/src/test/java/io/sphere/sdk/types/CustomFieldsGetterIntegrationTest.java

```diff
@@ -67,6 +67,11 @@ public void getFieldAsDouble() throws Exception {
     }
 
     @Test
+    public void getFieldAsBigDeciml() throws Exception {
```

Comment:
Would be nice if you could also add a test to `io.sphere.sdk.types.queries.WithCustomQueryModelIntegrationTest`.